### PR TITLE
add test that delete early error is recursive

### PR DESCRIPTION
**Pull request by devsnek**
i noticed this was missing after seeing all the tests for `delete ((this.#field))`

### CODE REVIEW

File: test/language/expressions/delete/identifier-strict-recursive.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2018 Mike Pennisi.  All rights reserved.
```
Comment (rwaldron) [inline test/language/expressions/delete/identifier-strict-recursive.js:1]: That's a funny way of spelling your name ;)

Comment (devsnek) [reply test/language/expressions/delete/identifier-strict-recursive.js:1]: this is copied from the other file in this directory, all i did was change the delete operand. should i change the copyright?

Comment (rwaldron) [reply test/language/expressions/delete/identifier-strict-recursive.js:1]: You created this file, so you should either add your name (such that both are there, on their own lines), or just change it to your name and this year.

Comment (devsnek) [reply test/language/expressions/delete/identifier-strict-recursive.js:1]: done

**Comment by rwaldron** [comment]
Thank you @devsnek

**Review by rwaldron — APPROVED** [review]

**Pull request merged**